Code/Editor/CryEdit.cpp

```diff
@@ -2322,6 +2331,62 @@ int CCryEditApp::IdleProcessing(bool bBackgroundUpdate)
     return res;
 }
 
+#if defined(CARBONATED)
+bool CCryEditApp::SetAIPause(bool paused)
+{
+    gEnv->pGame->GetIGameFramework()->SetAIPaused(paused);
+    return gEnv->pGame->GetIGameFramework()->IsAIPaused();
+}
+
+bool CCryEditApp::SetAICover(bool show)
+{
+    gEnv->pGame->GetIGameFramework()->SetAICover(show);
+    return gEnv->pGame->GetIGameFramework()->IsAICoverShown();
+}
+
+int CCryEditApp::SetCoverElev(int elev)
+{
+    gEnv->pGame->GetIGameFramework()->SetCoverElev(elev);
+    return gEnv->pGame->GetIGameFramework()->GetCoverElev();
+}
+
+int CCryEditApp::SetTimeScale(int inverseScale)
+{
+    gEnv->pGame->GetIGameFramework()->SetTimeScale(inverseScale);
+    return gEnv->pGame->GetIGameFramework()->GetTimeScale();
+}
+
+int CCryEditApp::SetAIImap(int imap)
+{
+    gEnv->pGame->GetIGameFramework()->SetAIImap(imap);
+    return gEnv->pGame->GetIGameFramework()->GetAIImap();
+}
+
+int CCryEditApp::SetAIImapSquad(int squad)
+{
+    gEnv->pGame->GetIGameFramework()->SetAIImapSquad(squad);
+    return gEnv->pGame->GetIGameFramework()->GetAIIMapSquad();
+}
+
+int CCryEditApp::SetAIDebugSquad(int squad)
+{
+    gEnv->pGame->GetIGameFramework()->SetAiSquadDebug(squad);
+    return gEnv->pGame->GetIGameFramework()->GetAiSquadDebug();
+}
+
+int CCryEditApp::SetAIDebugChar(int character)
+{
+    gEnv->pGame->GetIGameFramework()->SetAiCharDebug(character);
+    return gEnv->pGame->GetIGameFramework()->GetAiCharDebug();
+}
+
+int CCryEditApp::SetAIInputOverride(int team)
+{
+    gEnv->pGame->GetIGameFramework()->SetAiInputOverride(team);
+    return gEnv->pGame->GetIGameFramework()->GetAiInputOverride();
+}
+#endif
+
```

Comment:
This needs to be moved to our project., not the engine.  Please work with John A. and/or amazon to figure out how to get this into our project.